analyses/cell-type-wilms-tumor-06/notebook/README.md

```diff
@@ -0,0 +1,35 @@
+# Notebook directory instructions
+
+The notebook directory holds subdirectory for each of the sample in the Wilms tumor dataset SCPCP000006 and the fetal kidney reference that we used for label transfer. 
+Notebooks
+
+## Azimuth compatible fetal kidney reference
+
+To perform label transfer using Azimuth and the fetal kidney atlas, a reference is built via [`scripts/download-and-create-fetal-kidney-ref.R`](../scripts/download-and-create-fetal-kidney-ref.R) using the fetal_full.Rds object download from:
+"https://datasets.cellxgene.cziscience.com/40ebb8e4-1a25-4a33-b8ff-02d1156e4e9b.rds"
+
+As part of the `00b_characterize_fetal_kidney_reference_Stewart.Rmd` notebook template, we characterized the fetal kidney reference and generated lists of marker genes for the compartment and cell types composing the reference. 
+The tables of marker genes can be found in the S3 bucket name `researcher-008971640512-us-east-2`:
+- `references/00a_marker_cell-type_fetal_kidney_Stewart.csv`
+- `references/00a_marker_compartment_fetal_kidney_Stewart.csv` 
+
+The output can be found in the S3 bucket name `researcher-008971640512-us-east-2` named `references/ref.Rds` and `references/idx.annoy`.
```

Comment:
You don't need to include your bucket info here. If someone else needs these results, they will rerun your module (the researcher buckets are ONLY accessible by Data Lab for review).

```suggestion
```